src/util/transitTypeColors.scss

```diff
@@ -43,7 +43,3 @@
 .subwayBg {
   background-color: $subwayOrange !important;
 }
-
-.notFoundBg {
-  background-color: #000 !important;
```

Comment:
Why are all these set to !important?
We should avoid use !important, or only use !important if it is totally required.